pyanaconda/payload/dnf/payload.py

```diff
@@ -194,7 +194,12 @@ def get_source_proxy(self):
 
         :return: a DBus proxy
         """
-        return get_source(self.proxy, DNF_DEFAULT_SOURCE_TYPE)
+        default_source_type = DNF_DEFAULT_SOURCE_TYPE
+        # Default to the CDN if the if the Subscription
+        # DBus module is available.
+        if is_module_available(SUBSCRIPTION):
+            default_source_type = SOURCE_TYPE_CDN
```

Comment:
I thought that we will set `DNF_DEFAULT_SOURCE_TYPE` to `SOURCE_TYPE_CDN` on the rhel-8 branch. This is not a very good solution, because it breaks every logic that involves `DNF_DEFAULT_SOURCE_TYPE`.